### PR TITLE
Unfork from upstream

### PR DESCRIPTION
**Pull request by stvstnfrd**
This is a collection of miscellaneous changes to reduce our diff against upstream.

The contents here should be non-controversial.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@kluo @caesar2164 @dcadams  I went ahead and started addressing some of these. Larger changes will be contained in separate commits; this is for the small "why was this ever changed?" sort of cleanup..

**Comment by dcadams** [comment]
:+1:

File: common/lib/xmodule/xmodule/js/src/video/00_sjson.js [hunk 1]
```diff
@@ -62,6 +62,17 @@ function() {
...
+             * `start` and `end`. Start and end should
+             * be Numbers (doubles) corresponding to the
+             * number of seconds elapsed since the beginning
+             * of the video.
+             *
+             * Returns an object with properties
+             * "start" and "captions" representing
+             * parallel arrays of start times and
+             * their corresponding captions.
+             */
```
Comment (caesar2164) [inline common/lib/xmodule/xmodule/js/src/video/00_sjson.js]: why did we fork removing a comment????

**Comment by caesar2164** [comment]
👍

File: common/lib/xmodule/xmodule/js/src/video/00_sjson.js (hunk 1 above)
Comment (stvstnfrd) [inline common/lib/xmodule/xmodule/js/src/video/00_sjson.js]: I think it was a merge gone bad..

**Comment by stvstnfrd** [comment]
The only test failure is the quality check, but since we're unforking,
I'm going to ignore and merge as-is.

**Pull request merged**